src/integrationTest/java/uk/gov/hmcts/reform/divorce/orchestration/controller/IdamTestSupport.java

```diff
@@ -66,6 +68,13 @@
 
     static final String TOKEN_EXCHANGE_RESPONSE_1_JSON = convertObjectToJsonString(TOKEN_EXCHANGE_RESPONSE);
 
+
+    static final String USER_TOKEN = "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIwOTg3NjU0M"
```

Comment:
is it worth putting this with the other TestConstants?

Reply:
yeah can reuse then if needed in other places.